attendance/API_HR/c-hr.php

```diff
@@ -1737,18 +1737,25 @@ public static function API_getYearHolidays($year = false) {  //API
         $rows = self::DBfetchRows($runQuery);
         $list = array();
         
+        $type_text = self::getHolidayTypesList();
+
         if ($year == false) {
             $list = $rows;
         } else {
             foreach ($rows as $pp) {
                 $h_date = $pp['date'];
                 $h_year = date('Y', strtotime($h_date));
+                foreach($type_text as $text){
+                    if( $pp['type'] == $text['type'] ){
+                        $pp['text'] = $text['text'];
```

Comment:
try to use proper names of variables / keys etc, which should reflect what it is
change this to type_text